lib/gui/app.js

```diff
@@ -176,6 +176,11 @@ app.run(() => {
     ].join(' '));
 
     windowProgress.set(currentFlashState.percentage);
+
+    const DASH_UNICODE_CHAR = '\u2013';
+    const flashProgress = messages.info.flashProgress(currentFlashState);
+    const title = `${packageJSON.displayName} ${DASH_UNICODE_CHAR} ${flashProgress}`;
+    windowProgress.setTitle(title);
```

Comment:
IMHO it looks very odd to do `windowProgress.set()` and then `windowProgress.setTitle()`. IMHO either the `.set` should be renamed to something like `.setPercentage` or perhaps a better option is for the `.set` method itself to directly also update the title-bar?

EDIT: Which I guess would also have a nice symmetry with the `.clear` method directly updating the title-bar.